packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js

```diff
@@ -41,29 +52,37 @@ function SlideTutorial (props) {
             height={200}
             src={medium.src}
           />}
-        {activeStep === 0 &&
+        {isFirstStep &&
           <Heading level='3' margin={{ bottom: 'xsmall', top: 'small' }}>
             {counterpart('SlideTutorial.heading', { projectDisplayName })}
           </Heading>}
         {/* TODO: translation */}
         <Markdownz>{step.content}</Markdownz>
       </StyledMarkdownWrapper>
       <StepNavigation />
+      {isLastStep &&
+        <Button label={counterpart('SlideTutorial.getStarted')} onClick={onClick} margin={{ top: 'medium' }} primary />}
```

Comment:
List props here too? The linter might fix this, I'm not sure.

Reply:
I definitely forgot to run the linter. Will do.